### PR TITLE
add offset to date of 'riak-cs-gc status'

### PR DESCRIPTION
**Pull request by ksauzz**
`riak-cs-gc status`'s output looks like to need to convert epoch time to UTC.
#### before fix

```
% ./riak-cs-gc batch & ./riak-cs-gc status
[1] 6732
Garbage collection batch started.
A garbage collection batch is in progress
  The current garbage collection interval is: 900
  Last run started at: 00430917T132740Z                <------------
  Current run started at: 00430917T132742Z             <------------
  Next run scheduled for: 00430917T134240Z             <------------
  Elapsed time of current run: 0
  Files deleted in current run: 0
  Files skipped in current run: 0
  Files left in current run: 0
[1]  + 6732 done       ./riak-cs-gc batch
```
#### after fix

```
% ./riak-cs-gc batch & ./riak-cs-gc status
[1] 96703
Garbage collection batch started.
A garbage collection batch is in progress
  The current garbage collection interval is: 900
  Last run started at: 20130917T132621Z                <------------
  Current run started at: 20130917T132623Z             <------------
  Next run scheduled for: 20130917T134121Z             <------------
  Elapsed time of current run: 0
  Files deleted in current run: 0
  Files skipped in current run: 0
  Files left in current run: 0
[1]  + 96703 done       ./riak-cs-gc batch
```

### CODE REVIEW

File: src/riak_cs_gc_d.erl [hunk 1]
```diff
@@ -490,10 +490,9 @@ schedule_next(#state{interval=infinity}=State) ->
     State;
 schedule_next(#state{batch_start=Current,
                      interval=Interval}=State) ->
-    Next = calendar:gregorian_seconds_to_datetime(
-             riak_cs_gc:timestamp() + Interval),
+    Next = riak_cs_gc:timestamp() + Interval,
     _ = lager:debug("Scheduling next garbage collection for ~p",
-                    [Next]),
+                    [calendar:gregorian_seconds_to_datetime(Next)]),
```
Comment (shino) [inline src/riak_cs_gc_d.erl]: This log becomes "not offset" one, like `{{43,9,19},{7,59,2}}`.
How about deleting this debug log?

**Comment by shino** [comment]
+1 to merge after fixing above minor comment.

File: src/riak_cs_gc_d.erl (hunk 1 above)
Comment (ksauzz) [inline src/riak_cs_gc_d.erl]: Sure.

**Comment by ksauzz** [comment]
thanks.

**Pull request merged**